test_regress/t/t_assert_inside_cond_bad.out

```diff
@@ -1,3 +1,3 @@
-[10] %Error: t_assert_inside_cond.v:39: Assertion failed in top.t: synthesis parallel_case, but multiple matches found
+[10] %Error: t_assert_inside_cond.v:39: top.t: synthesis parallel_case, but multiple matches found
```

Comment:
These are assertions, we shouldn't change the output for these, only for info messages.

Reply:
Since we get a prefix of "%Error" I felt we can remove that "Assertion failed in" even for $error.

Reply:
@wsnyder please let me know if you want the original message for error and fatal or you are fine with the new style.

I will submit a new PR accordingly 

Thanks

Reply:
Thanks for that hint - used --push with fixes.